### PR TITLE
Return errors from `Event.GetData` and `Event.GetColumns`

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: processes/consumer/process.go [hunk 1]
```diff
@@ -60,7 +60,10 @@ func (p processArgs) process(ctx context.Context, cfg config.Config, inMemDB *mo
 	}
 
 	tags["op"] = _event.Operation()
-	evt := event.ToMemoryEvent(_event, pkMap, topicConfig.tc, cfg.Mode)
+	evt, err := event.ToMemoryEvent(_event, pkMap, topicConfig.tc, cfg.Mode)
+	if err != nil {
```
Comment (nathan-artie) [inline processes/consumer/process.go:64]: @Tang8330 Do we want to set `tags["what"]` here?

Comment (Tang8330) [reply processes/consumer/process.go:64]: `to_mem_event_err`?

**Comment by Tang8330** [comment]
This PR looks good, let's wait until next week to merge this since we're still seeing failures in logs

File: processes/consumer/process.go [hunk 2]
```diff
@@ -60,7 +60,11 @@ func (p processArgs) process(ctx context.Context, cfg config.Config, inMemDB *mo
...
+		tags["what"] = "to_mem_event_err"
+		return "", err
```
Comment (Tang8330) [inline processes/consumer/process.go:66]: Can we wrap this error?

**Review by Tang8330 — APPROVED** [review]

**Pull request merged**